### DISCUSSION TITLE
Achievements seem arbitrarily awarded

### DISCUSSION THREAD

**Discussion by seanmajorpayne** [community/community#27538] [Other Feature Feedback, Questions, & Ideas]
I have two Pull Shark awards. If the criteria is opening a pull request that gets merged, I should probably have hundreds of them assuming that an award is given for each merged pull request. Alternatively, I feel like it should only be 1 award since clearly this may be something that happens near daily.

Why specifically do I have two instances of the award? Is there something I'm missing here?

**Comment by radar-blip** [comment]
@seanmajorpayne It just updates at every tier (level) 2 - 16 - 128 - 1024

Info here: https://github.com/Schweinepriester/github-profile-achievements#achievements

**Comment by DenverCoder1** [comment]
This query shows you have ~70 merged pull requests which is why you are at the x2 level.

https://github.com/search?q=author%3Aseanmajorpayne+is%3Amerged+is%3Apr&type=issues

The base achievement is first awarded after 2 pull requests, a x2 (bronze) at your 16th pull request, x3 (silver) when you reach 128, and x4 (gold) at 1,024 merged pull requests.

Hovering over the achievement will tell you which pull requests earned you the tier you have reached. The repo in linuxlawson's answer (https://github.com/Schweinepriester/github-profile-achievements) will show you what you need to earn future achievements and tiers.

**Comment by jfullstackdev** [comment]
I have 3x of this 

![image](https://user-images.githubusercontent.com/47092464/181675891-2b60cda9-5615-4abb-afc9-62341eca4bc9.png)

it's not after every 2 but rather, 2 - 16 - 128
